platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.platform

```diff
@@ -472,7 +478,25 @@ connectivity named-singleton BLE {
 	 * The name of the device as advertised via GAP.
 	 */
 	configuration-item deviceName : string
-	
+	/**
+	 * Boolean representing if the MAC address is to be configured
+	 */
+	configuration-item isMACAddrConfigured : bool
```

Comment:
This will be removed. If MACAddr is set we update this. If not the code for MACAddr set-up will not be generated and the software will use a random MAC address.